build/common.go

```diff
@@ -76,33 +76,32 @@ func buildBackend(cfg Config) error {
 		return err
 	}
 
+	ldFlags := ""
+	if !cfg.EnableCGO {
+		// Link statically
+		ldFlags = `-extldflags "-static"`
+	}
+
+	if !cfg.EnableDebug {
+		// Add linker flags to drop debug information
+		ldFlags = "-w -s " + ldFlags
```

Comment:
Might be best to not add an extra space to the arguments, just in case it matters:
```suggestion
		prefix := ""
		if ldFlags != "" {
				prefix = " "
		}
		ldFlags = fmt.Sprintf("-w -s%s%s",  prefix, ldFlags)
```